hil/api.py

```diff
@@ -116,10 +117,15 @@ def project_detach_node(project, node):
 
     If the node has network attachments or pending network actions, a
     BlockedError will be raised.
+
+    If using a maintenance pool, please update the hil config accordingly.
     """
     project = _must_find(model.Project, project)
     get_auth_backend().require_project_access(project)
+    node_json = node
+    print(node_json)
```

Comment:
debugging statement? Also, isn't `node` just a string? Can you post what it looks like when you print it?

Reply:
I'm going to remove that print and replace the name with node_label.